### PR TITLE
Adds a loading indicator to PlansListViewController

### LINKED ISSUE #5173: Plans: add loading indicator
I have the code ready in a playground that was meant to tweak the animation values. It needs some polishing before adding it to the project:

- Set the right font/color for the title
- Use PDFs for the images

plans-loading

[PlansAnimation.playground_updated.zip](https://github.com/wordpress-mobile/WordPress-iOS/files/228129/PlansAnimation.playground_updated.zip)

### PR DESCRIPTION
**Pull request by frosty**
Fixes #5173. Adds the loading indicator from @koke's playground to `PlansListViewController`. I've added a small delay to the start of the animation so it doesn't begin until the VC has been pushed onscreen.

Needs review: @koke

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Plans/PlansLoadingIndicatorView.swift [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        circleView.addSubview(premiumView)
+        circleView.addSubview(freeView)
+        circleView.addSubview(businessView)
+        addSubview(circleView)
+
+        freeView.frame = targetFreeFrame
+        premiumView.frame = targetPremiumFrame
+        businessView.frame = targetBusinessFrame
+
+        circleView.backgroundColor = UIColor(red: 211/255, green: 222/255, blue: 230/255, alpha: 1)
```
Comment (koke) [inline WordPress/Classes/ViewRelated/Plans/PlansLoadingIndicatorView.swift]: I wonder if this should be a color from the style guide. The closest one seems to be `greyLighten20`

Comment (koke) [inline WordPress/Classes/ViewRelated/Plans/PlansLoadingIndicatorView.swift]: It looks like same color used as the background for the plan icons, so it's probably fine
screen shot 2016-05-16 at 08 20 06

**Comment by koke** [comment]
Looks good :shipit:

File: WordPress/Classes/ViewRelated/Plans/PlansLoadingIndicatorView.swift (hunk 1 above)
Comment (frosty) [inline WordPress/Classes/ViewRelated/Plans/PlansLoadingIndicatorView.swift]: Yeah, I took the color from the original Sketch document – there didn't seem to be anything particularly close in the style guide, so I just went with the actual color.

**Pull request merged**